docs/building-examples.rst

```diff
@@ -91,6 +91,16 @@ Enter the directory that contains examples for Xilinx 7-Series FPGAs:
 .. jinja:: xc7_linux_litex_demo
    :file: templates/example.jinja
 
+Additional Examples
+--------------------
+
+In addition to the designs we have gone over here, you can also find several other exciting designs 
```

Comment:
why do we need to add `additional examples` section? Why not simply extend the examples list?

Reply:
@kgugala I did do this with [`PR 166`](https://github.com/SymbiFlow/symbiflow-examples/pull/166). The reason I did not put all of my designs into the examples list was because I have 6 designs in total I would like to add to the examples. If I added all the designs to one document it would make the building examples page very long. I figured this would make things more organized.